testsuite/classlibrary/TestPattern.sc

```diff
@@ -155,6 +155,21 @@ TestPattern : UnitTest {
 		this.assert(x.sum { |x| x.delta } == 4.5, "Psync with maxdur = quant should end no earlier than after dur");
 	}
 
+	test_Pfset_nilStream {
```

Comment:
There is a unit testing guide somewhere -- one of the recommendations for test method names is that it should state actively what the test should do -- "test_what_action", e.g. `test_Pfset_evaluates_init_and_cleanup_on_empty_stream` -- which might seem pedantic, but it makes the test log unambiguous to read.

(It's here -- https://github.com/supercollider/supercollider/wiki/Unit-Testing-Guide -- but note that some points in the guide have been abandoned -- for instance, the guide says "one assert per test" but since then, multiple asserts have been recommended for clarity. So take with a grain of salt.)